qiita_db/user.py

```diff
@@ -509,6 +530,81 @@ def _change_pass(self, newpass):
             TRN.add(sql, [hash_password(newpass), self._id])
             TRN.execute()
 
+    def messages(self, count=None):
+        """Return messages in user's queue
+
+        Parameters
+        ----------
+        count : int, optional
+            Number of messages to return, starting with newest. Default all
+
+        Returns
+        -------
+        list of tuples
+            Messages in the queue, in the form
+            [(msg_id, msg, timestamp, read, system_message), ...]
+
+        Notes
+        -----
+        system_message is a bool. When True, this is a systemwide message.
+        """
+        with TRN:
+            sql_info = [self._id]
+            sql = """SELECT message_id, message, message_time, read,
+                     (expiration IS NOT NULL) as system_message
```

Comment:
`as` -> `AS` and add an extra indent here to be compliant with the new SQL coding guidelines.